devops/start.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env sh
```

Comment:
rename this file to `start_vscodedebug.sh` so its not mistaken for a regular 'start' script